config/environments/production.rb

```diff
@@ -51,9 +53,9 @@
   config.force_ssl = true
   config.ssl_options = { hsts: { preload: true } }
 
-  # Use the lowest log level to ensure availability of diagnostic information
-  # when problems arise.
-  config.log_level = :warn
+  # Include generic and useful information about system operation, but avoid logging too much
+  # information to avoid inadvertent exposure of personally identifiable information (PII).
+  config.log_level = :info
```

Comment:
I observe this changes our log level on prod (unclear on in which direction); do we want that?

Reply:
We do not, lemme fix it